Packs/Base/Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -1783,25 +1783,25 @@ def create_clickable_url(url):
 
 
 class JsonTransformer:
-    def __init__(self, flatten=False, keys_lst=None, is_nested=False, func=None):
+    def __init__(self, flatten=True, keys=None, is_nested=False, func=None):
         """
         Constructor for JsonTransformer
 
         :type flatten: ``bool``
         :param flatten:  Should we flatten the json using `flattenCell` (the default behavior)
 
-        :type keys_lst: ``List[str]``
-        :param keys_lst: a list of relevant keys list from the json
+        :type keys: ``List[str]``
+        :param keys: a list of relevant keys list from the json. Notice we save it as a set in the class
```

Comment:
then why not get it as a set?

Reply:
The initial reason was that Set is not JSON serializable and I wanted to use this class with `JsonToTable`.
But I think it's better just to pass an Iterable instead of list, so I will update it